example/passthrough_ll.c

```diff
@@ -87,6 +88,9 @@ static const struct fuse_opt lo_opts[] = {
 	  offsetof(struct lo_data, writeback), 1 },
 	{ "no_writeback",
 	  offsetof(struct lo_data, writeback), 0 },
+	{ "source=%s",
```

Comment:
I'd much rather see this as a parameter (like the mountpoint). Any reason you made it a mount option?